.github/workflows/peloton.yml

```diff
@@ -27,12 +33,9 @@ jobs:
       - name: "Set up environment"
         run: conda install -c conda-forge --file peloton/requirements.txt
 
-      - name: "Query GitHub"
-        env:
-          GITHUB_TOKEN: ${{ steps.generate_token.outputs.token }}
-        run: $CONDA/bin/python peloton/query.py
-
-      - name: "Populate Peloton Project"
-        env:
-          GITHUB_TOKEN: ${{ steps.generate_token.outputs.token }}
-        run: $CONDA/bin/python peloton/populate.py
+      - name: "Update the Peloton Project"
+        run: |
+          $CONDA/bin/python peloton/update_project.py \
+            --debug \
+            --bearer_token "${{ steps.generate_token.outputs.token }}" \
```

Comment:
@trexfeathers Is this the case?

Reply:
Unfortunately I can't tell from my demonstrations because I was unable to use the generating app in the `trexfeathers` space.

https://github.com/SciTools/.github/blob/737235479ec91f5a76484f1b85a44e5d17da1dbf/.github/workflows/peloton.yml#L23-L28

Instead I had to use a secret directly and GHA definitely knows to mask that out.

Actually: does the app generate a different token each time? In which case we wouldn't need to worry. I can't remember how it works.

Reply:
https://github.com/tibdex/github-app-token/issues/99 👀

Reply:
>Actually: does the app generate a different token each time? In which case we wouldn't need to worry. I can't remember how it works.

OK I think this is the case, since there is an opt-in to NOT revoke the token once the job finishes. So I'm no longer concerned.

I would be happy to convert over to the official GitHub action if you would prefer.